### PR TITLE
Feature/wa 35: text analytics component

### PR DESCRIPTION
**Pull request by MEGAFRAN**

### CODE REVIEW

File: app/components/text_analysis/TextAnalysis.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+          type="text"
+          placeholder={textAreaPlaceholder}
+          value={userMessage}
+          onChange={(event) => setUserMessage(event.target.value)}
+          aria-label={textAreaPlaceholder}
+          aria-invalid={!userMessage.trim()}
+          aria-describedby="user-message-error"
+          onInvalid={handleValidation}
+          onInput={handleInput}
+          required
```
Comment (Davidalbort) [inline app/components/text_analysis/TextAnalysis.tsx:61]: In line 20 there is a validation when the user clicked button submit and field input is empty, Could you valid why there are two valids, the first in line 20 and other this line?

File: app/components/text_analysis/TextAnalysis.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+          type="text"
+          placeholder={inputPlaceholder}
+          value={userMessageContext}
+          onChange={(event) => setUserMessageContext(event.target.value)}
+          aria-label={inputPlaceholder}
+          aria-invalid={!userMessageContext.trim()}
+          aria-describedby="user-message-context-error"
+          onInvalid={handleValidation}
+          onInput={handleInput}
+          required
```
Comment (Davidalbort) [inline app/components/text_analysis/TextAnalysis.tsx:76]: In line 20 there is a validation when the user clicked button submit and field input is empty, Could you valid why there are two valids, the first in line 20 and other this line?

File: app/components/text_analysis/TextAnalysis.tsx (hunk 1 above)
Comment (MEGAFRAN) [reply app/components/text_analysis/TextAnalysis.tsx:56]: you are right, these validations might be redundant

validation from line 20 was removed, individual validation message was kept on inputs

File: app/components/text_analysis/TextAnalysis.tsx (hunk 2 above)
Comment (MEGAFRAN) [reply app/components/text_analysis/TextAnalysis.tsx:71]: you are right, these validations might be redundant

validation from line 20 was removed, individual validation message was kept on inputs

**Pull request merged**